pandas/core/internals/managers.py

```diff
@@ -1455,7 +1461,7 @@ def __init__(self, block, axis, do_integrity_check=False, fastpath=False):
         if not isinstance(block, Block):
             block = make_block(block, placement=slice(0, len(axis)), ndim=1)
 
-        self.blocks = [block]
+        self.blocks = tuple([block])  # type: Tuple[Block]
```

Comment:
``BlockManager.blocks`` is a tuple, so this should also be a tuple, and not a list,